src/Compilers/CSharp/Portable/Syntax/LookupPosition.cs

```diff
@@ -289,24 +289,9 @@ private static SyntaxToken GetFirstIncludedToken(StatementSyntax statement, bool
                 case SyntaxKind.FixedStatement:
                     return ((FixedStatementSyntax)statement).FixedKeyword;
                 case SyntaxKind.ForEachStatement:
-                    // NB: iteration variable is only in scope in body.
-                    ForEachStatementSyntax forEachSyntax = (ForEachStatementSyntax)statement;
-                    if (inRecursiveCall)
-                    {
-                        return forEachSyntax.ForEachKeyword;
-                    }
-                    return GetFirstIncludedToken(forEachSyntax.Statement, inRecursiveCall: true);
+                    return ((ForEachStatementSyntax)statement).OpenParenToken.GetNextToken();
```

Comment:
I'm not sure this is right. The scope of the foreach variable is the embedded statement only, and does not include the expression after `in`. This change appears to modify that. For example, the following should print "Hello".

```cs
class Program
{
    static string c = "Hello";
    static void Main(string[] args)
    {
        foreach (char c in c.ToString())
        {
            Console.Write(c);
        }
        Console.WriteLine();
    }
}
```